### PR TITLE
Fix cases of making unnecessary nullable types

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

File: fastparquet/test/test_converted_types.py [hunk 1]
```diff
@@ -167,3 +167,13 @@ def test_tz_nonstring(tmpdir):
...
     assert (event_df == round).all().all()
+
+
+def test_pandas_simple_type(tmpdir):
+    import pandas as pd
+    fn = os.path.join(tmpdir, "out.parquet")
+    df = pd.DataFrame({"a": [1, 2, 3]}, dtype='uint8')
+    df.to_parquet(fn, engine="fastparquet")
+    df2 = pd.read_parquet(fn, engine="fastparquet")
+    assert df2.a.dtype == "uint8"
```
Comment (lithomas1) [inline fastparquet/test/test_converted_types.py:178]: The values should also be checked. If you want, you can use ``assert_frame_equal`` from pandas to check all the stuff at once.

**Review by lithomas1 — APPROVED** [review]
Thanks for the quick fix.

File: fastparquet/test/test_converted_types.py (hunk 1 above)
Comment (martindurant) [reply fastparquet/test/test_converted_types.py:178]: Can do - but the dtype is what is specifically being tested here.

Comment (lithomas1) [reply fastparquet/test/test_converted_types.py:178]: nbd. thanks again for the quick patch.

**Pull request merged**